.github/workflows/pipeline.yml

```diff
@@ -21,7 +21,9 @@ jobs:
         pip install --upgrade pip
         pip install "tox<4" "tox-gh-actions<3" "setuptools<58" "coveralls<4"
     - name: Test with tox
-      run: tox
+      run: |
+        mkdir .mypy_cache
```

Comment:
Is this only necessary for the `--install-types` argument? How about explicitly adding the necessary dependencies for typechecking, so these packages are also pinned?